packages/wd-button/src/WDButton.js

```diff
@@ -21,6 +22,179 @@ WDButton.STATE = {
    * @default example
    */
   id: Config.string().value('example'),
+
+  /**
+   * @type {?boolean}
+   * @default false
+   */
+  disabled: Config.bool().value(false),
+
+  /**
+   * @type {?(string|undefined)}
+   * @default undefined
+   */
+  elementClasses: Config.string(),
+
+  /**
+   * @type {?string}
+   * @default 'primary'
+   */
+  style: Config.oneOf(['primary', 'outline', 'ghost', 'danger', 'destructive', 'float']).value('primary'),
+
+  /**
+   * @type {?string}
+   * @default 'accent'
+   */
+  color: Config.oneOf(['accent', 'black', 'green', 'blue-light', 'blue', 'violet', 'red', 'orange', 'yellow']).value('accent'),
+
+  /**
+   * @type {?string}
+   * @default 'm'
+   */
+  size: Config.oneOf(['xs', 's', 'm', 'l', 'xl', 'xxl']).value('s'),
+
+  /**
+   * @type {?string|undefined}
+   * @default undefined
+   */
+  width: Config.string(),
+
+  /**
+   * @type {?boolean}
+   * @default false
+   */
+  hardWidth: Config.bool().value(false),
+
+  /**
+   * @type {?string}
+   * @default undefined
+   */
+  weight: Config.oneOf(['lightweight', 'middleweight', 'heavyweight']),
+
+  /**
+   * @type {?boolean}
+   * @default undefined
+   */
+  darkTheme: Config.bool(),
+
+  /**
+   * @type {?(string|undefined)}
+   * @default undefined
+   */
+  borderRadius: Config.oneOf(['borderRadius-6px', 'rounded']),
```

Comment:
please, create a file `radius` in `wd-base` that handles those border-radius

Reply:
@ygorcosta - I believe we will still need this borderRadius property on the button (to create a class) because the 'rounded' radiuses depend on the sizing (height) of the button. This also keeps the button object semantic.

Are you wanting me to simply add variables in wd-base for the values? Or are you wanting to get rid of this button object property?